### PR TITLE
Add /faq tournaments

### PR DESCRIPTION
**Pull request by AnnikaCodes**
Per the approved suggestion at https://www.smogon.com/forums/threads/add-a-tutorial-for-room-tours-as-a-faq.3661536/.

### CODE REVIEW

File: server/chat-commands/info.js [hunk 1]
```diff
@@ -1943,6 +1943,9 @@ const commands = {
 		if (showAll || target === 'rng') {
 			buffer.push(`<a href="https://${Config.routes.root}/pages/rng">How Pokémon Showdown's RNG works</a>`);
 		}
+		if (showAll || target === 'tournaments' || target === 'tournament' || target === 'tours' || target === 'tour') {
```
Comment (DaWoblefet) [inline server/chat-commands/info.js:1946]:
```suggestion
		if (showAll || ['tournaments', 'tournament', 'tours', 'tour'].indexOf(target) > -1) {
```

File: server/chat-commands/info.js [hunk 2]
```diff
@@ -1943,6 +1943,9 @@ const commands = {
...
+			buffer.push(`To join a room tournament, click the <b>Join!</b> button or type the command <code>/tour join</code> in the room's chat. You can check if your team is legal for the tournament by clicking the <b>Validate</b> button once you've joined and selected a team. To battle your opponent in the tournament, click the <b>Ready!</b> button when it appears. There are two different types of room tournaments: elimination (if a user loses more than a certain number of times, they are eliminated) and round robin (all users play against each other, and the user with the most wins is the winner).`);
```
Comment (DaWoblefet) [inline server/chat-commands/info.js:1947]:
```suggestion
			buffer.push(`To join a room tournament, click the <strong>Join!</strong> button or type the command <code>/tour join</code> in the room's chat. You can check if your team is legal for the tournament by clicking the <strong>Validate</strong> button once you've joined and selected a team. To battle your opponent in the tournament, click the <strong>Ready!</strong> button when it appears. There are two different types of room tournaments: elimination (if a user loses more than a certain number of times, they are eliminated) and round robin (all users play against each other, and the user with the most wins is the winner).`);
```
< strong > is preferred to < b > for bold representation in HTML5.

File: server/chat-commands/info.js (hunk 1 above)
Comment (Zarel) [reply server/chat-commands/info.js:1946]:
```suggestion
		if (showAll || ['tournaments', 'tournament', 'tours', 'tour'].includes(target)) {
```

**Comment by Zarel** [comment]
Works for me.

**Pull request merged**